src/js/components/action.jsx

```diff
@@ -17,6 +18,18 @@ export class ActionComponent extends Component {
         };
     }
 
+    onPostbackClick = () => {
+        this.setState({
+            state: 'processing'
+        });
+
+        postPostback(this.props._id).then(() => {
```

Comment:
Probably need a `catch` here to at least set the state to `done`, but we should do better error handling